aten/src/ATen/native/cpu/SortingKernel.cpp

```diff
@@ -19,6 +19,9 @@
 #ifdef USE_FBGEMM
 #include <fbgemm/Utils.h>
 #endif
+#if defined(__GNUC__) && !defined(__clang__) && defined(_OPENMP)
```

Comment:
Is it better to use ``#if __has_include(<parallel/algorithm>) && defined(_OPENMP)`` ?

Reply:
That's a really good suggestion. Done 👍